jquery.flot.touchNavigate.js

```diff
@@ -48,9 +48,6 @@ can set the default in the options. */
     'use strict';
 
     var options = {
-        pan: {
```

Comment:
I thought you fixed this already with the move of the code to use the processOptions hook